src/components/index.ts

```diff
@@ -0,0 +1,7 @@
+export {default as Card} from 'components/Card';
```

Comment:
Create index.ts files to export all components together to reduce import verbosity.